cmake/Dependencies.cmake

```
@@ -726,13 +726,15 @@ if(BUILD_PYTHON)
   endif()
 
   # Check that Python works
+  set(PYTHON_VERSION)
   if(DEFINED PYTHON_EXECUTABLE)
     execute_process(
         COMMAND "${PYTHON_EXECUTABLE}" "--version"
-        RESULT_VARIABLE _exitcode)
+        RESULT_VARIABLE _exitcode OUTPUT_VARIABLE _output_version)
```

- `python --version` actually outputs to stderr, not stdout.
- How about using `_python_version` instead of `_output_version`?

```suggestion
        RESULT_VARIABLE _exitcode ERROR_VARIABLE _output_version)
```

---

> 
> 
>     * `python --version` actually outputs to stderr, not stdout.
> 
>     * How about using `_python_version` instead of `_output_version`?

No, it outputs to stdout. I have tested this.

---

You are right about this. Then the suggested change should be only the about the variable name:

```suggestion
        RESULT_VARIABLE _exitcode OUTPUT_VARIABLE _python_version)
```